apps/dh/api-dh/source/DataHub.WebApi/Dto/MarketParticipant/Actor.cs

```diff
@@ -0,0 +1,29 @@
+﻿// Copyright 2020 Energinet DataHub A/S
+//
+// Licensed under the Apache License, Version 2.0 (the "License2");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+using System;
+using System.Collections.Generic;
+
+namespace Energinet.DataHub.WebApi.Dto.MarketParticipant
+{
+    public class Actor
+    {
+        public Guid Id { get; set; }
+
+        public string? ActorNumber { get; set; }
```

Comment:
ActorNumber is not optional.

Reply:
Fixed in https://github.com/Energinet-DataHub/greenforce-frontend/pull/1308/commits/a29ebbc8a6d7db8a553d64f1f3ceb1bd057f3e6d